api/v1alpha1/client/affiliate.proto

```diff
@@ -40,3 +41,8 @@ message Endpoint {
     bytes ip = 1;
     uint32 port = 2;
 }
+
+// ControlPlane optional configuration.
+message ControlPlane {
+    uint32 port = 1;
```

Comment:
I would make it more specific here, e.g. `api_server_port` (as there might be many more ports in the future)

Reply:
Will do!